salsa_dancing_molecules/simulations/simulation.py

```diff
@@ -1,4 +1,68 @@
 """Module for running a simulation."""
+from ..lennardjonesparse import parse_lj_params
+from ase.md.velocitydistribution import MaxwellBoltzmannDistribution
+from ase.md.verlet import VelocityVerlet
+from ase.md.andersen import Andersen
+from ase import units
+from asap3 import Trajectory
+from ..variables import Variables
+
+
+def choose_potential(potential, kim_model, use_asap, atoms):
+    """Change the atoms object according to what potential is used.
+
+    Args:
+        potential - String with name of potential to be used.
+        kim_model - String with ID for ensemble to be chosen from openkim.
+        use_asap - boolean if asap3 is to be used instead of ASE
+        atoms - atoms object to be edited.
+    """
+    potential = potential.lower().strip()
+    if potential == "openkim":  # use openkim
```

Comment:
Should we maybe print a warning if use_asap == True, but an openkim potential is given?